src/constants/mapboxStreetsV8Fields.ts

```diff
@@ -135,42 +31,227 @@ export const STREETS_V8_FIELDS = {
-  },
-
-  landuse_overlay: {
-    class: {
-      description: 'Overlay classification',
-      values: ['national_park', 'wetland', 'wetland_noveg'] as const
-    }
-  },
-
-  building: {
-    extrude: {
-      description: 'Should be extruded in 3D',
-      values: ['true', 'false'] as const
-    },
-    underground: {
-      description: 'Underground building',
-      values: ['true', 'false'] as const
     },
-    height: {
-      description: 'Building height',
-      type: 'number' as const
-    },
-    min_height: {
-      description: 'Building base height',
-      type: 'number' as const
+    type: {
+      description: 'type field',
+      values: [
+        'wood',
+        'farmland',
+        'forest',
+        'grass',
+        'meadow',
+        'scrub',
+        'parking',
+        'surface',
+        'park',
+        'farmyard',
+        'orchard',
+        'grassland',
+        'garden',
+        'school',
+        'soccer',
+        'vineyard',
+        'playground',
+        'tennis',
+        'bare_rock',
+        'allotments',
+        'pitch',
+        'heath',
+        'baseball',
+        'bunker',
+        'quarry',
+        'beach',
+        'basketball',
+        'scree',
+        'village_green',
+        'recreation_ground',
+        'sports_centre',
+        'common',
+        'christian',
+        'tee',
+        'sand',
+        'green',
+        'multi',
+        'hospital',
+        'greenhouse_horticulture',
+        'glacier',
+        'fairway',
+        'farm',
+        'golf_course',
+        'camp_site',
+        'university',
+        'college',
+        'plant_nursery',
+        'equestrian',
+        'fell',
+        'beachvolleyball',
+        'volleyball',
+        'american_football',
+        'athletics',
+        'caravan_site',
+        'rock',
+        'muslim',
+        'skateboard',
+        'wetland',
+        'bowls',
+        'picnic_site',
+        'boules',
+        'cricket',
+        'dog_park',
+        'running',
+        'conservation',
+        'track',
+        'netball',
+        'underground',
+        'lane',
+        'rugby_union',
+        'zoo',
+        'hockey',
+        'shooting',
+        'downhill',
+        'jewish',
+        'field',
+        'football',
+        'table_tennis',
+        'handball',
+        'rough',
+        'field_hockey',
+        'team_handball',
+        'carports',
+        'pelota',
+        'rugby',
+        'paddle_tennis',
+        'archery',
+        'horse_racing',
+        'gaelic_games',
+        'softball',
+        'golf',
+        'ice_hockey',
+        'basin',
+        'coastline',
+        'badminton',
+        'driving_range',
+        'bog',
+        'cricket_nets',
+        'swimming',
+        'futsal'
+      ] as const
     }
   },
 
-  water: {
-    // Water has no filterable fields
-  },
-
+  // ============ waterway ============
   waterway: {
+    iso_3166_1: {
+      description: 'iso_3166_1 field',
+      values: [
```

Comment:
[nit] The `iso_3166_1` list could be the same for each layer, consider puting them in one place, and referring to it, instead of listing it duplicate times.